src/main/java/entities/Topic.java

```diff
@@ -59,12 +88,52 @@ public boolean isInQueue(Member member) {
 
     /**
      * Remove and return the next Member in the queue.
-     * @return The Member at the front of the queue
+     * @return The new mentee
      */
     public Member getNextFromQueue() {
```

Comment:
Semantic : Generally when working with queues, you would use pop and put to adhere to naming conventions.